packages/cozy-harvest-lib/src/components/KonnectorConfiguration/ConfigurationTab/BIContractActivationWindow.spec.jsx

```diff
@@ -58,6 +58,7 @@ describe('BIContractActivationWindow', () => {
   })
 
   it('should display popup with url from policy and update contract after popup is closed', async () => {
+    expect.assertions(7)
```

Comment:
to be removed?

Reply:
No. Jest considers the test as success once at least an expectation is in success. However inside this long test, there is long tasks (waitFor) that are somehow skipped, because Jest continues testing other file. But the waitFor is still launched, outside its own test, and that provokes a warning (from jest-dom) making the whoe test suite failing.

Reply:
should I add more explanation inside a commit/code to justify this line?

Reply:
> No. Jest considers the test as success once at least an expectation is in success.

I really dont think this is the behavior. Are you sure? To me Jest considers the test as success once all the expected expectations are in success. There, I think the issue is because we have async except than Jest don't know about it. So we can say to jest "wait until you're call 7 times" but this is something we'll have to change every time we add something to this test. 

Did you try to return the promise inside the act & waitFor? 🤔

Reply:
thanks, that solution works